allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureAttachmentTest.java

```diff
@@ -0,0 +1,26 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Attachment;
+import org.junit.jupiter.api.Test;
+
+import java.nio.charset.StandardCharsets;
+
+public class AllureAttachmentTest {
+
+    @Test
+    public void annotatedAttachmentTest() {
+        textAttachment("Annotated", "hello, world!");
```

Comment:
the usage is pretty similar to `Allure.attachment("Dynamic attachment", "attachment content");` from the `dynamicAttachmentTest` below. It confuses a bit... What is meant under dynamism then? actually both calls look like being "dynamic", and in fact they are – dynamically called. 

I would at least consider renaming the "annotatedAttachmentTest" to something else... maybe `customDynamicAttachmentTest`

Reply:
UPDATE: probably this is too nerdy:) We can ignore this:)